packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -357,13 +353,13 @@ export function TransferPanelSummary({
         <>
           <div>
             <div className="h-2" />
-            <div className="lg:border-b lg:border-gray-3" />
+            <div className="border-b border-gray-7" />
```

Comment:
divider's border color is #EEEEEE (gray-3) according to Figma

Reply:
I am still confirming the design. #EEEEEE is not a part of the new gray specs. In fact, a lot of the grays we are using are not. We might need another activity to sync up these gray colors+tokens according to the design.

Reply:
Updated according to the latest design provided : 
![image](https://user-images.githubusercontent.com/7558499/235738747-b86ebc96-3ca9-497e-9ee0-8327b533651b.png)